Разметка всех страниц

verrtiga opened the pull request:
---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/34/user/2085107/tasks/4)

:boom: https://htmlacademy-htmlcss.github.io/2085107-sedona-34/2/

verrtiga committed:
Добавил скобки.

verrtiga committed:
Добавляет навигацию

verrtiga committed:
Обернул содежимое в <body>

verrtiga committed:
Добавляет секцию преимуществ

verrtiga committed:
Добавляет страницу about и hotels

verrtiga committed:
Добавляет section  и article

Добавил section с ссылкой на hotels.html  и article с формой для отправки email

verrtiga committed:
Добавляет footer

verrtiga committed:
Добавляет кнопку подписаться

verrtiga committed:
Добавил атрибут placeholder

Добавляет надпись в пустом поле ввода email

verrtiga committed:
Поменял названия файла с каталогом

verrtiga committed:
Добавляет радиокнопки и чекбоксы

verrtiga committed:
Добавил альт для хлебных крошек

verrtiga committed:
Добавляет секцию с фильтрами

verrtiga committed:
Выпадающий список и вид карточек товаров

Zakamsky commented on catalog.html:
```diff
@@ -0,0 +1,139 @@
+      <span class="visually-hidden"> Логотип Седона </span> 
+    </li>
+    <li class="navigation-item">
+       <a class="navigation-list" href="index.html"> Главная </a>
```
обычно используется просто  href="/" для главной. т.к. index.html это служебное имя.

Zakamsky commented on catalog.html:
```diff
@@ -0,0 +1,139 @@
+       <a class="navigation-list" href="index.html"> Главная </a>
+     </li>
+     <li class="navigation-item">
+       <a class="navigation-link" href="about.html"> О Седоне </a>
```
если ты не готов сверстать целую страницу с подвалом и шапкой, лучше сделать тут заглушку и удалить лишний файл...

Zakamsky commented on catalog.html:
```diff
@@ -0,0 +1,139 @@
+  </header>
+
+  <main class="main-catalog">
+  <header class="catalog-header">
```
Соблюдай отступы пожалуйста.

Zakamsky commented on catalog.html:
```diff
@@ -0,0 +1,139 @@
+    </fieldset>
+    <fieldset class="catalog-filter-group">
+    <button class="button" type="submit">Применить</button>
+    <input class="catalog-filter-reset" type="reset">
```
инпут это то куда мы вводим информацию, когда то давно для форм ими заменили кнопки. Но сейчас у нас есть отличный тэг кнопки и у него тоже есть `type="reset"`
Старайся избегать использования инпутов для кнопок, это уже устарело и имеет свои проблемы (например у инутов нет псевдоэлементов в большинстве браузеров)

Zakamsky commented on catalog.html:
```diff
@@ -0,0 +1,139 @@
+
+<section class="catalog">
+<b class="cataloge-title"> Найдено гостиниц: 38</b>
+<select class="cataloge-select" aria-label="Сортировка">
+  <option value="cheap"> Сначала дешевые </option>
+  <option value="popular"> Сначала популярные </option>
+  <option value="expensive"> Сначала дорогие</option>
+</select>
```
селект должен быть либьо частью формы, либо не быть.
форма должна иметь возмодность отправки.
Можно сделать этот элемент просто стилями, или использовать `details` посомтри в чате мы обсуждали.

Zakamsky commented on index.html:
```diff
@@ -5,8 +5,136 @@
+   <ul class="navigation-list">
+    <li class="navigation-logo "> 
+      <span class="visually-hidden"> Логотип Седона </span> 
+    </li>
```
ну это наверное все же картинка, а не спан....

Zakamsky reviewed (commented):
на вскидку - проверь на валидность есть одна грубая ошибка и несколько простых (в секциях нужны заголовоки)
кстати о заголовоках, в целом структура хороша, но есть один нюанс:
![image](https://user-images.githubusercontent.com/35491086/170031039-94ea0aa6-cbf3-447b-a60f-4716cf74bd21.png)
то что я выделил, разве это заголовоки того же уровня, или все же подзаголовки описания почему город настоящий? Лучше все же сделать их уровнем ниже (они и в разметке вложены на один уровень ниже. upd: увидел что нет, а надо бы, это ведь подразделы раздела, если это список то это подпункты первого пункта).

з.ы. сочетания списков хорошее, но пусть тебя не смущают номера - "упорядочнный список" запомни это название - это не список с циферками, это список в котором есть строгий порядок элементов, а их перестановка повлияет на их смысл. например хлебные крошки. или инструкция по заказу услуг - 1. вызываем замерщика 2. получаем счет 3.полачиваем 4.ждем доставку мы не можем переставить эти пункты в любом порядке - это упорядоченный список.

з.з.ы. в списки можно вкладывать только элементы списка `li`. вложенные списки вкладываются внутрь элементов списка  `li`

verrtiga committed:
Работа над ошибками

catalog.html
1. Поправил отступ
2. В 15 строке заменил index.html  на /
3. Поменял инпут, на баттон
4. Вставил селект в саму форму
index.html
1. Вставил в <li>  не порядковый список
2. Поменял структуру заголовков

verrtiga committed:
Исправил подвал и заголовки

verrtiga committed:
Завершил работу над катологом

Zakamsky commented on index.html:
```diff
@@ -5,8 +5,131 @@
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+    <section class="advantages">
+      <ol class="advanteges-list"> 
+        <li class="advantages-item first-item"> 
+          <h3 class="advantages-title"> Настоящий горододок </h3>
+        <p class="advantages-description"> Седона — не аттракцион для туристов, там течёт своя жизнь </p>
+        <img class="real-county" src="#" alt="Городок Седона">
+        <ul> 
+          <li class="advantages-item"> 
```
ну вот я опять очень долго пытаюсь понять, `h3` внутри `li` это верно, а `p`, `img` и `ul` рядом?

Zakamsky commented on catalog.html:
```diff
@@ -0,0 +1,244 @@
+    <fieldset class="catalog-filter-group">
+        <legend class="catalog-filter-title"> Инфраструктура </legend>
+        <ul class="catalog-filter-list">
+          <li class="catalog-filter-item">
+      <label class="catalog-filter-label">
+      <input class="catalog-filter-input" type="checkbox" name="swimming" checked>
+      Бассейн
+      </label>
+          </li>
```
что то с вложенностью совсем катастрофа какая то...

Zakamsky reviewed (commented):
форматирование кода вызывает у меня тоску и печаль, но в целом все корректно.
Я сейчас приму, но на защите у нас не примут селект отдельно от формы...
